File: js-learning-platzi/oop/Javascript/ecmas6/Car.js
class Car {
    constructor(license, driver){
        this.id,
        this.license = license,
        this.driver = driver,
        this.passenger
    }

    printDataCar(){
        console.log("Driver name: ", this.driver.name);
        console.log("Driver Document", this.driver.document);
        console.log("Lic: ", this.license);
    }
}

File: js-learning-platzi/oop/Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("car lic: " + license + " - Driver: " + driver.name);
    }
}